PovioKit/Classes/Extensions/Foundation/URL+Povio.swift

```diff
@@ -13,6 +13,17 @@ public extension URL {
     guard let string = string else { return nil }
     self.init(string: string)
   }
+  
+  /// Append parameter to the URL
+  func appending(_ name: String, value: String?) -> URL {
+    guard var components = URLComponents(url: self, resolvingAgainstBaseURL: true) else { return absoluteURL }
+    var queryItems = components.queryItems ?? []
+    let newQueryItem = URLQueryItem(name: name, value: value)
+    queryItems.append(newQueryItem)
+    components.queryItems = queryItems
+    components.percentEncodedQuery = components.percentEncodedQuery?.replacingOccurrences(of: "+", with: "%2B")
```

Comment:
Do we have any other symbols that would need to be encoded? 🤔

Reply:
Great question. I think we should have. Let me think about that.

Reply:
@nejcvivod it looks like components do its job fairly good. If I recall correctly I added percent handling for `+` because it isn't escaped by default 🤷‍♂